cli/flox-rust-sdk/src/models/pkgdb.rs

```diff
@@ -49,12 +55,9 @@ pub fn call_pkgdb(mut pkgdb_cmd: Command) -> Result<Value, CallPkgDbError> {
     let output = pkgdb_cmd.output().map_err(CallPkgDbError::PkgDbCall)?;
     // If command fails, try to parse stdout as a PkgDbError
     if !output.status.success() {
-        if let Ok::<PkgDbError, _>(pkgdb_err) = serde_json::from_slice(&output.stdout) {
-            Err(pkgdb_err)?
-        } else {
-            Err(CallPkgDbError::ParsePkgDbError(
-                String::from_utf8_lossy(&output.stdout).to_string(),
-            ))
+        match serde_json::from_slice::<PkgDbError>(&output.stdout) {
+            Ok(pkgdb_err) => Err(pkgdb_err)?,
```

Comment:
I'm now seeing:
```
> flox install python310Packages.conda
ERROR: error building environment: general error: general error: error:
           … while evaluating the attribute 'outPath'
```

Am I doing something wrong, or is that a bug in how we're printing the error in main?

Reply:
I'm taking a look locally, one sec

Reply:
See comment above, doesn't say anything about general error but that might be a similar issue

Reply:
From your comment it looks like the `category_message` was being printed twice because of the `: {0}`, but the `error:` here is probably a `context_message` that's providing...no context. This error should probably also fall under a different category than `general error` since the error specifically looks like a Nix evaluatation error.

Reply:
The `: {0}` syntax calls the `Display` trait impl for the type, and right now that's the `category_message` for `PkgDbError` if I remember correctly.

Reply:
Fixed, I needed to add a `context_message`